### PR TITLE
Update cache to use scoped packages

### PR DESCRIPTION
**Pull request by jarrodyellets**

### CODE REVIEW

File: lib/tutorials/en_US/caching.md [hunk 1]
```diff
@@ -108,20 +108,20 @@ hapi initializes a default [client](https://github.com/hapijs/catbox#client) usi
...
-const Hapi = require('hapi');
+const Hapi = require('@hapi/hapi');
 
 const server = Hapi.server({
     port: 8000,
     cache: [
         {
             name: 'mongoCache',
-            engine: require('catbox-mongodb'),
+            engine: require('@hapi/catbox-mongodb'),
```
Comment (Nargonath) [inline lib/tutorials/en_US/caching.md]: this one doesn't exist: https://www.npmjs.com/package/@hapi/catbox-mongodb

Comment (jarrodyellets) [reply lib/tutorials/en_US/caching.md]: Ahh, ok. I'm on it

Comment (Nargonath) [reply lib/tutorials/en_US/caching.md]: The repo has been archived apparently on GitHub: https://github.com/outmoded/catbox-mongodb. I guess it is not maintained anymore, isn't it?

Comment (jarrodyellets) [reply lib/tutorials/en_US/caching.md]: Doesn't look like it. I'll just remove it and keep catbox-redis.

Comment (Nargonath) [reply lib/tutorials/en_US/caching.md]: Nice

Comment (jarrodyellets) [reply lib/tutorials/en_US/caching.md]: forgot to lint....redo!

**Pull request merged**